wiki/Beatmap/Circle_size/sk.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - CS
+  - key count
+---
+
+# Veľkosť kruhov
+
+*Pre regulácie okolo veľkosti kruhov, pozri: [Kritéria celkového umiestnenia](/wiki/Ranking_Criteria)*
+
+**Veľkosť kruhov** (***CS***) je [beatmapové](/wiki/Beatmap) nastavenie obtiažnosti, ktoré mení veľkosť [trafiteľných objektov](/wiki/Hit_object). Hodnoty veľkosti kruhov majú rozsah od 0 do 10, ale iba hodnoty od 2 do 7 môžu byť vybraté v [editore beatmap](/wiki/Client/Beatmap_editor). Iné hodnoty môžu byť vybrané pomocou manuálnej zmeny [.osu súboru](/wiki/osu!_File_Formats/Osu_(file_format)) mapy.
+
+## osu!
+
+V osu!, veľkosť kruhov mení veľkosť kruhov a sliderov, kde väčšie hodnotu robia tieto objekty menšie. Spinnery nie sú afektované veľkosťou kruhov. Veľkosť kruhov sa vypočíta pomocou tohto vzorca:
+
+`r = 54.4 - 4.48 * CS`
+
+Where `r` is the radius measured in [osu!pixels](/wiki/osu!pixel), and `CS` is the circle size value.
```

Comment:
you forgot to translate this line

Reply:
ill be honest i have no idea how i missed it

time to fix it rq